submissions/ogennaOgbogu/htmlForms/signUp.html

```diff
@@ -0,0 +1,158 @@
+                        
+                        <div class="rightside">
+                            <label for="email">Email Address:</label>
+                            <br>
+                            <input type="email" class="text" name="email" id="email"/>
+                            <br>
+                        </div>
+                        
+                        <div class="rightside">
+                            <label for="password">Password:</label>
+                            <br>
+                            <input type="password" class="text" name="password" id="password"/>
+                        </div>
+                        
+                        <div class="rightside">
+                            <label for="password2">Confirm Password:</label>
+                            <br>
+                            <input type="password2" class="text" name="password2" id="password2"/>
+                            <br>
+                        </div>
+                        
+                        <div class="tandc">
+                            <div>
+                                <input type="checkbox" name="terms" id="terms"/>
+                                <label for="terms">I agree to the Terms and Conditions</label>
+                            </div>
+                        </div>
+                        
+                        <br>
+
+                        <div class="button1">
+                        <button type="button" id="signupbutton">SIGN UP</button>
+                        </div>
+                        </div>
+                    </form>
+                </div>   
+        </div>           
+            
+
+            <!--footer-->
+        <footer class="footer">
+                        <div class="aboutus">
+                            <p><a href="#">Contact Us</a></p>
+                            <p><a href="#">About Us</a></p>
+                            <p><a href="#">Terms and Conditions</a></p>
+                        </div>     
+
+                        <div class="rightfooter">
+
+                            <div class="candc">
+                                <div class="contact">
+                                    <label for="email">Sign up for our Newsletter</label>
+                                    <br>
+                                        <div class="newsletter">
+
+                                            <input type="text" name="newsletter" id="newsletter" placeholder="Enter your email address" required/>
+                                        <button type="button" id="submitbutton">SUBMIT</button>
+                                        </div>
+                                        
+                                </div> 
+                            </div>      
+
+                            <div class="social-network">
+                                <p>CONNECT WITH US</p>
+                                <div class="social-img">
```

Comment:
@Oge-nna Style the images in your css

Reply:
Hi @ToluAkin, I also don't understand, style in what way? Nest them in hyperlinks or? Please explain.

Reply:
Oh. Nvm. I understand now. Thank you.